scripts/scraper-ng/rules/html-require-recipe-ingredients/ingredient-handlers/utils.js

```diff
@@ -49,6 +49,39 @@ function sliceBetween(startNode, endCondition, tree) {
   return newRoot;
 }
 
+/**
+ * Find the first node that you didn't expect to find, or return `null`.
+ *
+ * @param {Object} tree - the tree to search for unexpected nodes
+ * @param {Array} expectedTrees - An array of nodes such that each node itself
+ *   and all of its descendants are permited. An expected tree's descendants are
+ *   never visited.
+ * @param {Array} expectedNodes - An array of nodes such that each node is
+ *   permitted. An expected node's descendants are visited, if they exist. The
+ *   starting node and white space are always expected.
+ * @returns {Object|null} a node or `null`
+ */
+function findExtraneousNode(tree, expectedTrees, expectedNodes) {
```

Comment:
This is really nitpicky but "Unexpected" might be a clearer word than "Extraneous".

Reply:
✅ Renamed `findExtraneousNode` to `findUnexpectedNode`